instance/github.py

```diff
@@ -158,13 +159,8 @@ def __init__(self, number, source_fork_name, target_fork_name, branch_name, titl
         self.title = title
         self.username = username
         self.body = body
-
-    @property
-    def extra_settings(self):
-        """
-        Extra settings contained in the PR body
-        """
-        return get_settings_from_pr_body(self.body)
+        self.extra_settings = get_settings_from_pr_body(body)
+        self.extra_settings_dict = yaml.load(self.extra_settings) or {}
```

Comment:
Although here a middle way could be to set `extra_settings` as a new attribute, and remove `pr_body` altogether, or turn it into a method or setter. You're right that it probably makes more sense to standardize the data structure around `extra_settings`, which would be used by both PRs instances and other types of instances, rather than `pr_body` which would only be used by PR instances.